### DISCUSSION TITLE
Running LocalGPT on any cloud with SkyPilot

### DISCUSSION THREAD

**Discussion by romilbhardwaj** [PromtEngineer/localGPT#218] [Show and tell]
Thanks for this really awesome project! I put together a short guide on how to run LocalGPT and the UI on any cloud with just 1 click using SkyPilot - https://github.com/skypilot-org/skypilot/blob/master/llm/localgpt/README.md

Hope this is useful to someone trying to run LocalGPT on the cloud.

**Comment by PromtEngineer** [comment]
@romilbhardwaj this is awesome, thanks for sharing. It definitely helpful for a lot of folks.

**Comment by maxng07** [comment]
Thank you Romil for sharing. This is helpful for my upcoming project.
